R/input-utils.R

```diff
@@ -3,6 +3,8 @@ shinyInputLabel <- function(inputId, label = NULL) {
     label,
     class = "control-label",
     class = if (is.null(label)) "shiny-label-null",
+    # `id` attribute is required for `aria-labelledby` used by screen readers:
+    id = paste0(inputId, "-label"),
```

Comment:
By any chance, do you have any reference that would explain why the `id` is needed (in addition to the `for` below)?